test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Microsoft.VisualStudio.LanguageServices.Razor.Test.csproj

```diff
@@ -14,6 +14,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="WindowsBase" />
```

Comment:
What's this for?

Reply:
This is where `System.Windows.Threading.Dispatcher` lives